README.md

```diff
@@ -1,148 +1,188 @@
-# What is Crate?
+# What Is Crate?
 
-Crate is _the_ distributed database for containerized environments, such as Docker.
-Based on the familiar SQL syntax, Crate combines high availability, resiliency, and scalability
-in a distributed design that allows you to query mountains of data in realtime, not batches.
+Crate is an open source fast, scalable, easy to use SQL database that plays
+nicely with containers like Docker. It feels like the SQL databases you know,
+but makes scaling and operating your database ridiculously easy - regardless of
+the volume, complexity, or type of data. It ingests millions of records per
+second for time series setups and delivers analytics results in sub-second real
+time.
 
-We solve your data scaling problems and make administration a breeze.
+Crate comes with a distributed sort and aggregation engine, fast multi index
+queries, native full-text search and super simple scalability with sharding and
+partitioning builtin. Preconfigured replication takes care of data resiliency.
+The cluster management can be supervised with a built-in admin UI. Crate's
+masterless architecture and simplicity make the data part of Docker environments
+easy and elegant.
 
-**Crate.IO - Put your data to work. Simply.**
+Crate provides several installation packages, including a supported Docker
+image. It fits perfectly into an orchestrated microservices environment. It acts
+like an ephemeral, omnipresent, persistent layer for data. Application
+containers access their data regardless of which host the data nodes run.
 
-[http://crate.io][3]
+[Crate](https://crate.io/)
 
 
-# Crate Dockerfile
+![logo](https://raw.githubusercontent.com/docker-library/docs/2517900006ae5f4c03c1d43235930c59f4614394/crate/logo.png)
 
-This repository contains **Dockerfile** of [Crate][3] for [Docker][1]'s [automated build][2]
-published to the public [Docker Hub Registry][4].
+## How To Use This Image
 
+To form a cluster, start a few instances of the Crate container as a background
+daemon:
 
-## Base Docker Image
+```console
+# docker run -d crate/crate
```

Comment:
i'd start first with the full command first - so that it makes sense to run crate, and then go to the easier cases to explain. :) ```docker run -d crate/crate -d -v ...:/data -p 4200... -p 4300... --env CRATE_HEAP_SIZE=32g``` probably even passing -Des... parameters.